webpack.config.js

```diff
@@ -52,17 +52,6 @@ module.exports = {
                     },
                 ],
             },
-            {
-                test: /\.(jpe?g|gif|png|svg)$/i,
-                use: [
-                    {
-                        loader: 'url-loader',
```

Comment:
Is `file-loader` still being used? If so, let's create an issue to use [asset modules](https://webpack.js.org/guides/asset-modules/) since `file-loader` was deprecated for webpack v5.